saas/static/js/djaodjin-saas-angular.js

```diff
@@ -621,6 +621,102 @@ transactionControllers.controller("userRoleDescriptionCtrl",
     }
 }]);
 
+subscriptionControllers.controller("planSubscribersListCtrl",
```

Comment:
It might be possible to declare/create this controller on the model of `subscriberListCtrl ` (saas/static/js/djaodjin-saas-angular.js):

    subscriptionControllers.controller("subscriberListCtrl",
        ["$scope", "$controller", "$http", "$timeout", "settings",
        function($scope, $controller, $http, $timeout, settings) {
        settings.urls.saas_api_subscriptions = settings.urls.saas_api_active_subscribers;
        $controller('subscriptionListCtrl', {
            $scope: $scope, $http: $http, $timeout:$timeout,
            settings: settings});
    }]);

Otherwise this pull request looks really good. Thank you!